eth/package.json

```diff
@@ -64,6 +64,8 @@
     "node": ">=16"
   },
   "scripts": {
+    "hh": "hardhat",
+    "hardhat": "hardhat",
```

Comment:
Do we want these in this PR?

Reply:
no haha. I forgot about npx so I was using these.
```suggestion
```